plugins/hydra_ray_launcher/example/aws/conf/config.yaml

```
@@ -0,0 +1,16 @@
+defaults:
+  - hydra/launcher: ray_aws
+
+hydra:
+  launcher:
+    params:
+      code_sync_cfg:
+        code_dir: ""
+        include: ["model", "*.py"]
+        exclude: ["*"]
```

1. Since this is an example, add comments describing any nontrivial config node.
2. Is code_dir relative? to what?
```suggestion
      code_sync:
        source_root: null
        include: [model, *.py]
        exclude: [*]
```